### PR TITLE
Removes most sec reskins so that its all one style instead of 3 conflicting ones

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
image
**Xyel approved this PR - having such a hodge-podge of styles for whats supposed to be an organized department is confusing, lore-unfriendly, and honestly just doesnt look nice.**

What this PR does is actually mostly just cleanup: unused/redundant security sprites/items are axed (I hate Junior Officer), reskins are fixed (our version and the TG version use the same variable, so when our version is 'disabled' the TG version tries to do it anyways. All this causes are errors.), Peacekeeper alts re-removed (Security is new now, having oldblue and newblue contributes to the whole 'mismatching' thing), and Red skins are removed where applicable (No more skirts for Warden/HoS!(HoS is the only one allowed to keep Red for formality sake, but they have Blue stuff now too where applicable))
image
(This image doesnt really show how the reskins are fixed, but just trust me on that; also the HoS has a blue alt for his cap now, shown here, which is nice :)
And ignore the draft medic sprites. Please.)

Security medic gets blue coats now. It actually matches pretty well with the red & white. ENTIRELY AN OPTIONAL CHOICE.
image
Credit to halcyon for the recolor <3

It also moves the ENTIRE secvendor list to modular (yes, the redsec vendor. It helps with a conflict we had concerning the peacekeeper one being a subtype, so it'd add the redsec 'skyrat_products') and adds redsec items as [item]/redsec for admin events, vendors, or the likes. Easy to search, too.
image

---
### A VERY big note: 
**The one single master icon file does NOT work. Instead, I've begun transitioning us to a new system, where we have master sub-files for clothes, just like TG does; this one adds the Security.dmi's, and puts all clothes into them that should be there.**
The current method of cramming everything into one file just isnt maintainable. Just LOOK at the file. You cant tell whats TG and whats Skyrat, and you cant tell where Dresses start and where Costumes end. This can NOT stay this way, as it leads to sprites that arent updated/maintained (Digi captain sprites which still used the old palette, for example), or sprites no longer in use just clogging it up further (Peacekeeper1, Peacekeeper2 - why did we need  or 5 alts of old peacekeeper stuff?)

## How This Contributes To The Skyrat Roleplay Experience
Better sprite maintainability, more consistent security, less reskin jank, updates some sprites, removes lots of unused sprites/items.

## Changelog

:cl: Orion_the_Fox, Halcyon(Sprites)
del: Armadyne and Lopland have agreed to remove OldPeacekeeper reskins and some straggling Redsec clothes from Security, after several Uniformity complaints from CC.
imageadd: SPLIT THE MASTER UNIFORM FILE TO HAVE A SECURITY SUBTYPE. ONLY SECURITY UNIFORMS SHOULD BE IN HERE. EXPECT THIS TO EXPAND ONTO OTHER DEPARTMENTS AS WELL. (TG already does this, its much easier to maintain than one giant file.)
imagedel: Removed lots of unused Security icons. (Literally, these had no items. Ingame you'll see no change from this.)
/:cl:

### CODE REVIEW

File: code/modules/vending/wardrobes.dm [hunk 1]
```diff
@@ -15,24 +15,26 @@
 	icon_state = "secdrobe"
 	product_ads = "Beat perps in style!;It's red so you can't see the blood!;You have the right to be fashionable!;Now you can be the fashion police you always wanted to be!"
 	vend_reply = "Thank you for using the SecDrobe!"
+	/* LISTS OVERRIDDEN IN 'modular_skyrat\modules\sec_haul\code\misc\vending.dm'
```
Comment (ghost) [inline code/modules/vending/wardrobes.dm]:
```suggestion
	/* SKYRAT EDIT - LISTS OVERRIDDEN IN 'modular_skyrat\modules\sec_haul\code\misc\vending.dm'
```

File: modular_skyrat/modules/customization/modules/clothing/under/misc.dm [hunk 2]
```diff
@@ -83,13 +83,6 @@
 	armor = list(MELEE = 30, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 10, ACID = 20, WOUND = 10)
 	body_parts_covered = CHEST|GROIN|ARMS|LEGS
 
-/obj/item/clothing/under/misc/discounder
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/under/misc.dm:86]: was added upstream, no longer needed here

File: modular_skyrat/modules/customization/modules/clothing/under/security.dm [hunk 3]
```diff
@@ -1,16 +1,6 @@
-// MODULAR SECURITY WEAR
+// MODULAR SECURITY WEAR (NOT OVERRIDES, LOOK IN 'modular_skyrat\modules\goofsec\code\sec_clothing_overrides.dm')
 
 // HEAD OF SECURITY
-
-/obj/item/clothing/under/rank/security/head_of_security/parade
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/under/security.dm:5]: sprite was added to the upstream parade uniform, this is no longer needed

File: modular_skyrat/modules/customization/modules/clothing/under/security.dm [hunk 4]
```diff
@@ -25,45 +15,21 @@
...
 	desc = "A white satin shirt with a leather belt, the belt buckle is a large NT."
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/security.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/security.dmi'
 	icon_state = "policechiefalt"
 
 // DETECTIVE
-
-/obj/item/clothing/under/rank/security/detective/undersuit
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/under/security.dm:34]: this and the bomber were added upstream

File: modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm [hunk 5]
```diff
@@ -480,3 +489,100 @@
...
+	worn_icon = 'icons/mob/clothing/head/winterhood.dmi'
+	icon_state = "hood_security"
+
+/obj/item/clothing/suit/hooded/wintercoat/security/red
+	icon = 'icons/obj/clothing/suits/wintercoat.dmi'
+	worn_icon = 'icons/mob/clothing/suits/wintercoat.dmi'
+	icon_state = "coatsecurity"
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/security/red
+
+//PLASMA MEN ARE TO BE LEFT ALONE!!!
```
Comment (ghost) [inline modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm]:
```suggestion
```
moving to top of file

File: modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm [hunk 6]
```diff
@@ -1,5 +1,6 @@
 /////////
 // Overwrites all the security icons with our own bluesec versions; this means little to no mapping/spawning conflicts!
+// (Also includes 'old' versions at the bottom of the file, for the purpose of crates/vendors/admin shenanigans. Can't remove them ENTIRELY after all!)
```
Comment (ghost) [inline modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm:3]:
```suggestion
// (Also includes 'old' versions at the bottom of the file, for the purpose of crates/vendors/admin shenanigans. Can't remove them ENTIRELY after all!)
//PLASMA MEN ARE TO BE LEFT ALONE!!! ARMADYNE/LOPLAND DONT MAKE PEACEKEEPER PLASMASUITS!
```

File: modular_skyrat/modules/sec_haul/code/misc/vending.dm [hunk 7]
```diff
@@ -22,7 +22,7 @@
 		/obj/item/storage/fancy/donut_box = 2
 	)
 	premium = list(
-		/obj/item/storage/belt/security/webbing = 5,
+		/obj/item/storage/belt/security/webbing/peacekeeper = 5, //SKYRAT EDIT CHANGE - added /peacekeepr
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/misc/vending.dm]:
```suggestion
		/obj/item/storage/belt/security/webbing/peacekeeper = 5, //SKYRAT EDIT CHANGE - added /peacekeeper
```
typo moment

also the reason I'm doing a big review of my own file is bc its easier to sort through all the changes in this, so uh, dont mind me

**Comment by ghost** [comment]
I'll undraft once the secmed cursed sprite is solved, already looking into it

**Comment by ghost** [comment]
now that i have sec medic sprites
i await review

File: _maps/map_files/Blueshift/BlueShift_middle.dmm [hunk 8]
```diff
@@ -6550,8 +6550,7 @@
 /obj/item/clothing/under/rank/security/head_of_security/formal,
 /obj/item/clothing/under/rank/security/head_of_security/parade,
 /obj/item/clothing/under/rank/security/head_of_security/parade/female,
-/obj/item/clothing/suit/armor/hos/parade,
-/obj/item/clothing/suit/armor/hos/parade/female,
+/obj/item/clothing/suit/armor/hos/hos_formal = 1,
```
Comment (ghost) [inline _maps/map_files/Blueshift/BlueShift_middle.dmm:6553]:
```suggestion
/obj/item/clothing/suit/armor/hos/hos_formal,
```
wtf

File: modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa/ghost_spawn.dm [hunk 9]
```diff
@@ -33,13 +33,13 @@
 	outfit = /datum/outfit/security_guard
 	you_are_text = "You are a security guard in a top secret government facility. You blacked out. Now, you have woken up to the horrors that lay within. DO NOT TRY TO EXPLORE THE LEVEL. STAY AROUND YOUR AREA."
 
-/obj/item/clothing/under/rank/security/peacekeeper/junior/sol/blackmesa
+/obj/item/clothing/under/rank/security/peacekeeper/sol/traffic/blackmesa
```
Comment (Iamgoofball) [inline modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa/ghost_spawn.dm:36]: why is this pathed like this? this is some /obj/structure/bed/chair/stool stuff

File: modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm [hunk 10]
```diff
@@ -1,5 +1,7 @@
...
+//PLASMA MEN ARE TO BE LEFT ALONE!!! ARMADYNE/LOPLAND DONT MAKE PEACEKEEPER PLASMASUITS!
```
Comment (Iamgoofball) [inline modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm:4]: we should just sort out the plasmamen outfits already

File: modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm [hunk 11]
```diff
@@ -480,3 +490,99 @@
...
+	worn_icon = 'icons/mob/clothing/eyes.dmi'
+	icon_state = "securityhud"
+	glass_colour_type = /datum/client_colour/glass_colour/red
+
+/obj/item/clothing/glasses/hud/security/sunglasses/redsec
+	icon = 'icons/obj/clothing/glasses.dmi'
+	worn_icon = 'icons/mob/clothing/eyes.dmi'
+	icon_state = "sunhudsec"
+	glass_colour_type = /datum/client_colour/glass_colour/darkred
+	current_skin = "sunhudsec"	//prevents reskinning; a bit hacky to say its already reskinned but its better than a code rewrite
```
Comment (Iamgoofball) [inline modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm:579]: this is not the correct way to go about this

File: modular_skyrat/modules/customization/modules/clothing/under/security.dm [hunk 12]
```diff
@@ -97,7 +52,38 @@
 	mutant_variants = NONE
 	can_adjust = FALSE
 
-/// SEC GENERAL
+///CDO
```
Comment (Iamgoofball) [inline modular_skyrat/modules/customization/modules/clothing/under/security.dm]: CDO doesn't exist, just axe this

File: modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa/ghost_spawn.dm (hunk 9 above)
Comment (ghost) [reply modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa/ghost_spawn.dm:36]: I'm just removing the /junior, the others are all subtypes of the peacekeeper and im hesitant to touch that many itempaths in a PR thats meant to just remove redundant shit (which is, agian, the /junior)

File: modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm (hunk 10 above)
Comment (ghost) [reply modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm:4]: I'm getting it next PR dont worry ;)

File: modular_skyrat/modules/customization/modules/clothing/under/security.dm (hunk 12 above)
Comment (ghost) [reply modular_skyrat/modules/customization/modules/clothing/under/security.dm]: Its there for the police uniforms that once belonged to CDO and that I'm not removing bc its used for other stuff, but yeah I'll change the comment at the least

File: modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm (hunk 11 above)
Comment (ghost) [reply modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm:579]: The other solution is to set uses_advanced_reskins = FALSE and set list/unique_reskin = null, but means twice as much code thats also fairly copy-paste-y

File: modular_skyrat/modules/customization/modules/clothing/under/security.dm (hunk 12 above)
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/under/security.dm]:
```suggestion
///Police Uniforms
```

File: modular_skyrat/modules/customization/modules/clothing/under/security.dm [hunk 13]
```diff
@@ -97,7 +52,38 @@
...
+	desc = "A light blue shirt with navy pants, perfect for opressing the Underclasses, like Catgirls."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/security.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/security.dmi'
+	icon_state = "policealt"
+
+///WARDEN
+/obj/item/clothing/under/rank/security/warden/peacekeeper/sol
+	name = "sol warden uniform"
+	desc = "A light blue shirt with navy pants, this one seems to have been modified for fat asses, like yourself."
+	icon_state = "policewardenalt"
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/under/security.dm:77]:
```suggestion
	icon_state = "policewardenalt"
	
// End Police Uniforms
```

**Comment by hugmeorelse** [comment]
Red security is not coming back.

I explicitly stated this. Don't use my words to push a PR like this.

**Comment by hugmeorelse** [comment]
Thank you for working on this, however, there are a few issues.

Firstly, we have no intention of bringing back red security. Having it as a vendor is redundant and should not be available for players. Also, HoS should not have access to it. I would not have it available. 

Also, you are removing items that are integral to the style we put out which we'll need to sort out. This PR should not be merged until these concerns are addressed. 

**Edit:** Github be acting funky.

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]
You've removed things that aren't supposed to be removed. Please convene in the discord. Until this is sorted out, this is DNM.

**Comment by ErdinyoBarboza** [comment]
you removed normal variant of webbings which is the +1 slot webbing and replaced it with the peacekeeper variant
please re-add it back above the peacekeeper variant

File: modular_skyrat/modules/sec_haul/code/misc/vending.dm [hunk 14]
```diff
@@ -22,7 +22,7 @@
 		/obj/item/storage/fancy/donut_box = 2
 	)
 	premium = list(
-		/obj/item/storage/belt/security/webbing = 5,
+		/obj/item/storage/belt/security/webbing/peacekeeper = 5, //SKYRAT EDIT CHANGE - added /peacekeeper
```
Comment (MassacreDemigirl) [inline modular_skyrat/modules/sec_haul/code/misc/vending.dm]:
```suggestion
		/obj/item/storage/belt/security/webbing = 5,
		/obj/item/storage/belt/security/webbing/peacekeeper = 5, //SKYRAT EDIT CHANGE - added /peacekeeper
```

Comment (MassacreDemigirl) [reply modular_skyrat/modules/sec_haul/code/misc/vending.dm]: You removed the +1 item capacity premium webbing. Please make sure to re-add it

**Review by MassacreDemigirl — CHANGES_REQUESTED** [review]

**Comment by Ebin-Halcyon** [comment]
> Firstly, we have no intention of bringing back red security. Having it as a vendor is redundant and should not be available for players. Also, HoS should not have access to it. I would not have it available.

Redsec vendors are admin spawn only. They aren't in any maps and aren't available to collect/get anywhere.

**Comment by ghost** [comment]
> 

Not true. Until this is merged at least.
Currently they spawn in lots of checkpoints.

**Comment by Ebin-Halcyon** [comment]
> > 
> 
> Not true. Until this is merged at least. Currently they spawn in lots of checkpoints.

Meant, what the PR does, dumdum.

**Comment by ghost** [comment]
> > > 
> > 
> > 
> > Not true. Until this is merged at least. Currently they spawn in lots of checkpoints.
> 
> Meant, what the PR does, dumdum.

oh we sorted it out in DMs in that case, dont worry :)

**Comment by Gandalf2k15** [comment]
Reopen once requested changes have been made.

File: modular_skyrat/modules/customization/modules/clothing/suits/coats.dm [hunk 15]
```diff
@@ -212,6 +212,10 @@
 	toggle_noun = "zipper"
 	armor = list(MELEE = 10, BULLET = 5, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 0, ACID = 0)
 
+/obj/item/clothing/suit/toggle/brit/sec/worn_overlays(mutable_appearance/standing, isinhands, icon_file)
+	. = ..()
+	if(!isinhands)
+		. += emissive_appearance(icon_file, "[icon_state]-emissive", alpha = src.alpha)
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/suits/coats.dm]:
```suggestion
```
Do it in another PR you dork, it'll still be there

File: modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm [hunk 16]
```diff
@@ -109,6 +109,11 @@
 	icon_state = "labcoat_highvis"
 	mutant_variants = NONE
 
+/obj/item/clothing/suit/toggle/labcoat/highvis/worn_overlays(mutable_appearance/standing, isinhands, icon_file)
+	. = ..()
+	if(!isinhands)
+		. += emissive_appearance(icon_file, "[icon_state]-emissive", alpha = src.alpha)
+
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm]:
```suggestion
```
Do it in another PR, it'll still be there

File: modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm [hunk 17]
```diff
@@ -360,10 +397,6 @@
 		"White Variant" = list(
 			RESKIN_ICON_STATE = "vest_white",
 			RESKIN_WORN_ICON_STATE = "vest_white"
```
Comment (ghost) [inline modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm]:
```suggestion
			RESKIN_WORN_ICON_STATE = "vest_white"
		),
		"Peacekeeper Variant" = list(
			RESKIN_ICON_STATE = "peacekeeper_armor",
			RESKIN_WORN_ICON_STATE = "peacekeeper"
```
Resprite means this fits now, we can leave it

File: modular_skyrat/modules/sec_haul/code/misc/vending.dm [hunk 18]
```diff
@@ -62,22 +63,38 @@
 					/obj/item/storage/backpack/satchel/sec = 3,
 					/obj/item/storage/backpack/duffelbag/sec = 3,
 					/obj/item/clothing/under/rank/security/officer = 10,
+					/obj/item/clothing/under/rank/security/peacekeeper/tactical = 4,
 					/obj/item/clothing/shoes/jackboots/security = 10,
 					/obj/item/clothing/head/security_garrison = 10,
 					/obj/item/clothing/head/security_cap = 10,
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/misc/vending.dm:69]:
```suggestion
					/obj/item/clothing/head/security_cap = 10,
					/obj/item/clothing/head/beret/sec/peacekeeper = 5,
```

File: modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm [hunk 19]
```diff
@@ -30,28 +38,16 @@
...
 	icon_state = "security_medic_turtleneck"
 
 /obj/item/clothing/under/rank/security/peacekeeper/security_medic/skirt
 	name = "security medics's skirtleneck"
-	desc = "A comfy red turtleneck with a white armband and brown skirt, denoting the wearer as a security medic."
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
-	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/uniform_digi.dmi'
+	desc = "A comfy turtleneck with a white armband and brown skirt, denoting the wearer as a security medic."
 	icon_state = "security_medic_turtleneck_skirt"
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm:58]:
```suggestion

/obj/item/clothing/under/rank/security/peacekeeper/security_medic/old
	name = "security medics's uniform"
	desc = "A lightly armored uniform worn by Nanotrasen's Asset Protection Medical Corps."
	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/uniform_digi.dmi'
	icon_state = "security_medic_jumpsuit"

```
Security medic is getting this all cleaned out separately because its even more of a mess

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm [hunk 20]
```diff
@@ -113,14 +114,12 @@
...
+	new /obj/item/clothing/suit/toggle/labcoat/security_medic/blue(src)
+	new /obj/item/clothing/suit/hazardvest/security_medic/blue(src)
 	new /obj/item/clothing/head/helmet/sec/peacekeeper/security_medic(src)
-	new /obj/item/clothing/under/rank/security/peacekeeper/tactical(src)
 	new /obj/item/radio/headset/headset_medsec(src)
 	new /obj/item/storage/medkit/emergency(src)
 	new /obj/item/clothing/suit/straight_jacket(src)
 	new /obj/item/storage/belt/medical(src)
 	new /obj/item/storage/belt/security/medic/full(src)
 	new /obj/item/clothing/under/rank/medical/doctor/red(src)
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm]:
```suggestion
	new /obj/item/clothing/under/rank/medical/doctor/red(src)
	new /obj/item/clothing/under/rank/security/peacekeeper/security_medic/old
```
It can stay while security medic has a full overhaul

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm [hunk 21]
```diff
@@ -57,9 +57,9 @@
 	ears = /obj/item/radio/headset/headset_medsec
 	uniform = /obj/item/clothing/under/rank/security/peacekeeper/security_medic
 	gloves = /obj/item/clothing/gloves/color/latex/nitrile
-	shoes = /obj/item/clothing/shoes/jackboots
+	shoes = /obj/item/clothing/shoes/jackboots/security
 	glasses = /obj/item/clothing/glasses/hud/health/sunglasses
-	suit = /obj/item/clothing/suit/armor/vest/peacekeeper/security_medic
+	suit = /obj/item/clothing/suit/toggle/labcoat/security_medic/blue
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm]:
```suggestion
	suit = /obj/item/clothing/suit/armor/vest/peacekeeper/security_medic
```
Medic rework is another PR

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm [hunk 22]
```diff
@@ -113,10 +114,9 @@
 
 /obj/structure/closet/secure_closet/security_medic/PopulateContents()
 	..()
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:101]:
```suggestion
	..()
	new /obj/item/clothing/suit/toggle/labcoat/security_medic(src)
	new /obj/item/clothing/suit/hazardvest/security_medic(src)
```
may as well give the choice though, good way to get some player input

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm [hunk 23]
```diff
@@ -113,10 +114,9 @@
...
-	new /obj/item/clothing/suit/toggle/labcoat/security_medic(src)
-	new /obj/item/clothing/suit/hazardvest/security_medic(src)
+	new /obj/item/clothing/suit/toggle/labcoat/security_medic/blue(src)
+	new /obj/item/clothing/suit/hazardvest/security_medic/blue(src)
 	new /obj/item/clothing/head/helmet/sec/peacekeeper/security_medic(src)
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:106]:
```suggestion
	new /obj/item/clothing/head/helmet/sec/peacekeeper/security_medic(src)
	new /obj/item/clothing/under/rank/security/peacekeeper/tactical(src)
```

File: modular_skyrat/modules/sec_haul/code/misc/vending.dm [hunk 24]
```diff
@@ -62,22 +63,39 @@
...
-					/obj/item/clothing/gloves/color/black/security = 10,
-					/obj/item/storage/belt/security/webbing/peacekeeper = 2,
-					/obj/item/storage/backpack/security/peacekeeper = 5,
-					/obj/item/storage/backpack/satchel/sec/peacekeeper = 5,
 					/obj/item/clothing/head/beret/sec/peacekeeper = 5,
-					/obj/item/clothing/under/rank/security/peacekeeper = 5,
-					/obj/item/clothing/under/rank/security/peacekeeper/tactical = 2,
+					/obj/item/clothing/gloves/color/black/security = 10,
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/sec_haul/code/misc/vending.dm:71]: Keep the tactical outfit and the backpack, since they are favoured options

Comment (ghost) [reply modular_skyrat/modules/sec_haul/code/misc/vending.dm:71]: tactical outfit is just moved up in the vendor to be in the same spot as all the other Under items
